configuration/concepts/concepts_base.csv

```diff
@@ -0,0 +1,15 @@
+Uuid,Void/Retire,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,_version:1,_order:1000
+0aca2d79-1a16-42c8-b2c0-a4989bd55aec,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,Drug,N/A,,
+48728efa-b311-4696-a8d0-f2bc155ccd35,,ABC+3TC+ATZ/r ,,ABC+3TC+ATZ/r ,,ABC+3TC+ATZ/r ,,Drug,N/A,,
```

Comment:
@IipperMangesh Please ensure that there is no trailing space after the concept's name. Could you please also trim the concept names on the lines below as well?